app/controllers/tutorials_controller.rb

```diff
@@ -1,19 +1,9 @@
 class TutorialsController < ApplicationController
   def index
-    tutorials
+    @tutorials = Tutorial.includes(:tutorial_steps).all
   end
 
   def show
-    tutorial
-  end
-
-  private
-
-  def tutorial
-    @tutorial ||= Tutorial.find(params[:id])
-  end
-
-  def tutorials
-    @tutorials ||= Tutorial.all
+    @tutorial = Tutorial.includes(:tutorial_steps).find(params[:id])
```

Comment:
does this execute the find first? e.g.

select * from tutorials where id = <:id>;
select * from tutorial_steps where tutorial_id = <:id>;

Reply:
Yep! The "after" screenshot in the PR description shows that behavior.